fdbserver/LogRouter.actor.cpp

```diff
@@ -217,6 +231,8 @@ ACTOR Future<Void> waitForVersion( LogRouterData *self, Version ver ) {
 	if(ver >= self->startVersion + SERVER_KNOBS->MAX_VERSIONS_IN_FLIGHT) {
 		self->foundEpochEnd = true;
 	}
+	self->waitForVersionTime += now() - startTime;
```

Comment:
The variable name seems a bit misleading: the name indicates it is the single wait time. 
A better name can be `totalWaitForVersionTime`.